### PR TITLE
[MAT-6195] Display calculation results (QDM)

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
## MADiE PR

Jira Ticket: [MAT-6195](https://jira.cms.gov/browse/MAT-6195)
(Optional) Related Tickets:

### Summary
Once calculation has been run, display a `Result` block containing the Calculation Result. 

The result will contain the `statement_result.[cqlLib].[definition].pretty` value from cqm-execution's response.

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [x] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx [hunk 1]
```diff
@@ -133,21 +142,33 @@ const QdmGroupCoverage = ({
       }, {});
   };
 
+  const getDefinitionCalcResult = (definitionName) => {
+    const popCriteriaResults =
+      calculationResults[Object.keys(calculationResults)[0]][selectedCriteria];
+    let calcResult;
+    Object.keys(popCriteriaResults.statement_results).forEach((cqlLib) => {
+      calcResult ??=
```
Comment (RohitKandimalla) [inline src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx:150]: ??= thats an interesting operator, and I can't seem to google its purpose.

Comment (RohitKandimalla) [reply src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx:150]: and if we found calcResult, why are we not returning.
if (calcResult) return;

Comment (jkotanchik-SB) [reply src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx:150]: Here, it prevents `calcResult` form being overwritten once its set. Admittedly, it's probably not needed since it will return once set, short-cruiting the loop.

> only evaluates the right operand and assigns to the left if the left operand is [nullish](https://developer.mozilla.org/en-US/docs/Glossary/Nullish) (null or undefined).

See: [MDN Nullish Coalescing Assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment)

Comment (RohitKandimalla) [reply src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx:150]: Thats good to know.

Comment (RohitKandimalla) [reply src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx:150]: I think I understood it, Looks like we are getting the calcResult for the definitionName found in the first library and ignoring all the others. 

I can't figure out the whole picture here, 
but Can we trust that the first element/key of `Object.keys(popCriteriaResults.statement_results) `is always the appropriate library that we need ?

**Review by RohitKandimalla — DISMISSED** [review]

**Review by RohitKandimalla — APPROVED** [review]

**Comment by RohitKandimalla** [comment]
I thought I approved it
